src/plugins/support/InvocationSupport.ts

```diff
@@ -0,0 +1,48 @@
+import ThundraLogger from '../../ThundraLogger';
+
+class InvocationSupport {
```

Comment:
What about exporting get/set tag methods directly as function in the `InvocationSupport.ts` instead of providing through `InvocationSupport` class. 

I mean, for example, user can just import `setTag` method from `InvocationSupport` js file and use it without `InvocationSupport.getInstance()` since tag related operations are just utility functionalities. @hamitb did similar approach for Python agent

Reply:
Ok. I made all of `InvocationSupport` methods static.  User need not need to call `InvocationSupport.getInstance()` any more. They can directly call methods statically.